test/repository_test.rb

```diff
@@ -68,6 +68,16 @@
             UserRepository.persist(persisted_user)
             persisted_user.created_at.must_equal persisted_user.created_at
           end
+
+          it 'update updated_at' do
+            update_user = UserRepository.find(user.id)
+            update_user.name = 'My'
+
+            UserRepository.persist(update_user)
```

Comment:
I don't think so. Because `UserRepository.persist(update_user)` it updates an entity. I want to get `updated_user` when I fetch an entity from the collection by its ID.